EmbyStat.Common/Models/Entities/Show.cs

```diff
@@ -16,6 +16,8 @@ public class Show : Extra
         public List<Season> Seasons { get; set; }
         [BsonRef(nameof(Episode))]
         public List<Episode> Episodes { get; set; }
+        public double Size { get; set; }
```

Comment:
In the entities classes, we only want properties that are saved to the database. Size and MissingEpisodes are not saved because they can be calculated. So I'm not going to add them

Reply:
You got me thinking, maybe it's better to add them, after all, 😄 let me check if that's possible, it would speed up the calculations probably!